Things.md

```diff
@@ -169,4 +170,10 @@ Events can also be added to repeat, similar to using `setInterval`.  A new third
 FSM.addEventInterval(FSM.shiftHoriz, 100, 5, thing, 2);
 ```
 
-More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
\ No newline at end of file
+More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
+
+# Thing classes
+
+Classes are implemented using [ObjectMakr](https://github.com/FullScreenShenanigans/ObjectMakr), a module that automates the creation of the inheritance structure of classes. The structure and a list of properties are given and the class objects are then constructed using this information. 
```

Comment:
> are given and the

It would be better to say that it takes them in when initialized. It's vague to say phrases like "are given" because it's not clear *when* they're given.